docs/get/README.md

```diff
@@ -0,0 +1,120 @@
+
+Open your favorite browser and navigate to `http://localhost:19999` or `http://REMOTE-HOST:19999` to open the dashboard.
+
+### Other operating systems/methods
+
+Want to install Netdata on a Kubernetes cluster, with Docker, or using a different method? Not a Linux user? Choose your
+platform to see specific instructions.
+
+<Install>
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/kubernetes"
+    img="/img/index/methods/kubernetes.svg"
+    os="Kubernetes" />
+  <InstallBox
+    to="/docs/agent/packaging/docker"
+    img="/img/index/methods/docker.svg"
+    os="Docker" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/cloud"
+    img="/img/index/methods/cloud.svg"
+    os="Cloud providers (GCP, AWS, Azure)" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/packages"
+    img="/img/index/methods/package.svg"
+    os="Linux with .deb/.rpm" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/kickstart-64"
+    img="/img/index/methods/static.svg"
+    os="Linux with static 64-bit binary" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/manual" 
+    img="/img/index/methods/git.svg"
+    os="Linux from Git" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/freebsd"
+    img="/img/index/methods/freebsd.png"
+    os="FreeBSD" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/macos"
+    img="/img/index/methods/macos.svg"
+    os="MacOS" />
+</Install>
+
+Even more options available in our [packaging documentation](/docs/agent/packaging/installer#alternative-methods).
+
+## Claim your node on Netdata Cloud
+
+You need to [claim](/claim/README.md) your nodes to see them in Netdata Cloud. Claiming establishes a secure TLS
+connection to Netdata Cloud using the [Agent-Cloud link](/aclk/README.md), and proves you have write and administrative
+access to that node.
+
+When you view a node in Netdata Cloud, the Agent running on that node streams metrics, metadata, and alarm status to
+Netdata Cloud, which in turn streams those metrics to your web browser. Netdata Cloud does not store or log metrics or
+alarm status.
+
+To claim a node, you need to run the claiming script. In Netdata Cloud, click on your Space's name, then **Manage your
+Space** in the dropdown. Click **Nodes** in the panel that appears. Copy the script and run it in your node's terminal.
+For example:
+
+```bash
+sudo netdata-claim.sh -token=TOKEN -rooms=ROOM1,ROOM2 -url=https://app.netdata.cloud
+```
+
+The script should return `Agent was successfully claimed.` after creating a new RSA pair and establishing the link to
+Netdata Cloud.
+
+For more information on the claiming process, why we implemented it, and how it works, see the [claim](/claim/README.md)
+and [Agent-Cloud link](/aclk/README.md) reference docs.
+
+## What's next?
+
+You've now set up your free Netdata Cloud account, installed the Agent, and claimed one or more nodes to your Space.
```

Comment:
claimed --> claim

Reply:
I think _claimed_ is still correct, as I'm referencing actions that are (hopefully) in the past. I did rephrase the sentence slightly though.